packages/chrome-extension/src/popup/AllPuzzles.tsx

```diff
@@ -149,7 +149,9 @@ export default class AllPuzzles extends React.Component<Props, State> {
     }
 
     filterBySearchInputValue = (puzzle: IPuzzle) => {
-        return puzzle.name.toLowerCase().indexOf(this.state.searchInputValue.toLowerCase()) !== -1;
+        const query = this.state.searchInputValue.toLowerCase();
+        return puzzle.name.toLowerCase().indexOf(query) !== -1 ||
+            (puzzle.solution && puzzle.solution.toLowerCase().indexOf(query) !== -1);
```

Comment:
we really shouldn't have `solution` set if `status !== "solved"`, but I'd probably check anyway
```suggestion
            (puzzle.status === "solved" && puzzle.solution && puzzle.solution.toLowerCase().indexOf(query) !== -1);
```